### PR TITLE
New MAKES Interpret example

### PR DESCRIPTION
**Pull request by DarrinEide**
Example shows reference parsing, same as the new Project Academic Knowledge example

### CODE REVIEW

File: academic-services/knowledge-exploration-service/reference-get-interpret.md [hunk 1]
```diff
@@ -456,3 +456,84 @@ Date: Thu, 30 Jan 2020 01:08:52 GMT
...
+
+```
+Sinha, Arnab, et al. "An Overview of Microsoft Academic Service (MAS) and Applications." Proceedings of the 24th International Conference on World Wide Web, 2015, pp. 243–246.
+```
+
+#### Request headers
+
+```http
+GET /interpret?query=Sinha,%20Arnab,%20et%20al.%20"An%20Overview%20of%20Microsoft%20Academic%20Service%20(MAS)%20and%20Applications."%20Proceedings%20of%20the%2024th%20International%20Conference%20on%20World%20Wide%20Web,%202015,%20pp.%20243–246.&count=1&entityCount=1&attributes=Id,DN,Y,AA.DAuN,VFN HTTP/1.1
+Host: makesexample.westus.cloudapp.azure.com
```
Comment (RickRogahn) [inline academic-services/knowledge-exploration-service/reference-get-interpret.md]: >makesexample.westus.cloudapp.azure.com [](start = 6, length = 38)

guessing this is fake?

Comment (DarrinEide) [reply academic-services/knowledge-exploration-service/reference-get-interpret.md]: Yes, it's the same one I use for all MAKES examples

---
In reply to: [398024088](https://github.com/MicrosoftDocs/microsoft-academic-services/pull/135#discussion_r398024088) [](ancestors = 398024088)

File: academic-services/project-academic-knowledge/reference-interpret-method.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 title: Interpret method
 description: Generates semantic interpretations of a natural language query
 ms.topic: reference
-ms.date: 2020-03-19
+ms.date: 2020-03-25
```
Comment (RickRogahn) [inline academic-services/project-academic-knowledge/reference-interpret-method.md]: >2020-03-25 [](start = 9, length = 10)

FYI - Have you seen the build warnings for our date format?  I think they want them in mm/dd/yyyy format now...

Comment (DarrinEide) [reply academic-services/project-academic-knowledge/reference-interpret-method.md]: How very American of them.

---
In reply to: [398027232](https://github.com/MicrosoftDocs/microsoft-academic-services/pull/135#discussion_r398027232) [](ancestors = 398027232)

**Pull request merged**